dataeng/jobs/createJobs.groovy

```diff
@@ -0,0 +1,30 @@
+import static analytics.AnalyticsEmailOptin.job as AnalyticsEmailOptinJob
+import static analytics.AnalyticsExporter.job as AnalyticsExporterJob
```

Comment:
nit:   why does github highlight the characters on either side of " as " on these two lines?  If they were just spaces I wouldn't expect a red square.

Reply:
Not sure what's going on, files created using vim and github show same behavior. I found this: https://github.com/textmate/groovy.tmbundle/issues/17 which made me think it was textmate related(I use textmate). But doesn't look like that it is. olivex internal dsl repo show the same red blocks.